easy-coding-standard.yml

```diff
@@ -3,8 +3,15 @@ imports:
     - { resource: 'packages/*/easy-coding-standard.yml' }
     - { resource: 'project-base/easy-coding-standard.yml' }
 
+services:
```

Comment:
is this change necessary? it is already declared in the imported config from `project-base` :thinking: